master/docs/relnotes/index.rst

```diff
@@ -22,11 +22,14 @@ Features
 
 * :class:`GitPoller` now has a ``buildPushesWithNoCommits`` option to allow the rebuild of already known commits on new branches.
 * Add GitLab authentication plugin for web UI. See :class:`buildbot.www.oauth2.GitLabAuth`.
+* :class:`DockerLatentWorker` now has a ``hostconfig`` parameter that can be used to setup host configuration when creating a new
```

Comment:
In RST "one line = one sentence" convention is used in Buildbot (not everywhere yet), please place whole sentence on a single line.